test/azul_test_case.py

```diff
@@ -189,6 +189,7 @@ def _reset_moto(self):
         catalog: config.Catalog(name=catalog,
                                 atlas='hca',
                                 internal=False,
+                                needs_slots=True,
```

Comment:
nit. might be safer to have these test catalog attributes set to False, just in case someone is writing tests for the azulclient.....